dds_web/api/user.py

```diff
@@ -1254,3 +1254,62 @@ def get_users(unit: models.Unit = None):
             "keys": keys,
             "empty": not users_to_return,
         }
+
+
+class InvitedUsers(flask_restful.Resource):
+    """Provide a list of invited users."""
+
+    @auth.login_required()
+    def get(self):
+        out_model = ("email", "role", "created_at", "public_id")
+        current_user = auth.current_user()
+        if current_user.role == "Super Admin":
+            raw_hits = models.Invite.query.all()
+        elif current_user.role in ("Unit Admin", "Unit Personnel"):
+            unit = models.UnitUser.query.filter_by(username=current_user.username).one().unit
+            owned_projects = models.Project.query.filter_by(unit_id=unit.id).all()
+            unit_projects = set(proj.id for proj in owned_projects)
```

Comment:
Slight simplification
```suggestion
            unit = current_user.unit
            unit_projects = set(proj.id for proj in unit.projects)
```